### DISCUSSION TITLE
Simple getting started documentation for react

### DISCUSSION THREAD

**Discussion by tigawanna** [Tencent/cherry-markdown#574] [Q&A] [answered]
Hey guys , this looks like the best markdown editor I've tried but I have no idea on how to get started in a react app 

Can anyone help with a snippet of how to initialize?

**Comment by RSS1102** [comment, marked as answer]
https://github.com/RSS1102/cherry-markdown-react-demo
Hello, you can refer to this repo. Thank you for your suggestion. In the future, we will also add react/vue templates in the [examples](https://github.com/Tencent/cherry-markdown/tree/main/examples).

**Reply by tigawanna** [reply]
Thank you for the quick response .
This would also be awesome for frameworks like svelte, solid and qwik which don't have that many markdown editors

**Reply by RSS1102** [reply]
ok，In the future, basic templates will be developed for some mainstream front-end frameworks. Thank you very much again for your affirmation.
